goalgorilla/open_social
# Issue #3177350 by agami4: Fix display of button to send a private message on /user/inbox page

## Problem
There is no button to send a private message on the `/user/inbox` page

## Solution
Update styles for the `region--complementary` class

## Issue tracker
https://www.drupal.org/project/social/issues/3177350

## How to test
- Go to the `/user/inbox` page`
- Open inspector element and find `New message` text
- Check position current button

## Screenshots
before:
<img width="1623" alt="Screenshot 2020-10-16 at 17 14 00" src="https://user-images.githubusercontent.com/16086340/96274494-37989b00-0fd9-11eb-80db-0b8d059305a9.png">

after:
<img width="1415" alt="Screenshot 2020-10-16 at 17 46 17" src="https://user-images.githubusercontent.com/16086340/96273201-bc82b500-0fd7-11eb-8c07-307e475998fa.png">

## Release notes
The `New message` button on the `/user/inbox` page will have a correct position

## Change Record
-

## Translations
-
